core/templates/services/exploration-html-formatter.service.spec.ts

```diff
@@ -38,18 +38,18 @@ describe('Exploration Html Formatter Service', () => {
     ehfs = TestBed.get(ExplorationHtmlFormatterService);
   });
 
-  it('should correctly set interaction HTML for TextInput when it is in' +
-     ' editor mode', () => {
-    var interactionId = 'TextInput';
+  it('should correctly set interaction HTML for a non migrated interaction ' +
+     'when it is in editor mode', () => {
+    var interactionId = 'nonMigratedInteraction';
     let custArgs = {
       placeholder: {value: new SubtitledUnicode('enter here', '')},
       rows: {value: 1}
     };
-    var expectedHtmlTag = '<oppia-interactive-text-input ' +
+    var expectedHtmlTag = '<oppia-interactive-non-migrated-interaction ' +
       'placeholder-with-value="{&amp;quot;unicode_str&amp;quot;:&amp;quot;' +
       'enter here&amp;quot;,&amp;quot;content_id&amp;quot;:&amp;quot;&amp;' +
       'quot;}" rows-with-value="1" last-answer="lastAnswer">' +
-      '</oppia-interactive-text-input>';
+      '</oppia-interactive-non-migrated-interaction>';
```

Comment:
@DubeySandeep, I was not able to find the previous comment in the sea of comments 😛 . I have made the change we discussed offline.

Reply:
np, thanks for the fix! :)